packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -685,8 +709,7 @@ export type HooksTree = Array<HooksNode>;
 // of a hook call. A simple way to demonstrate this is wrapping `new Error()`
 // in a wrapper constructor like a polyfill. That'll add an extra frame.
 // Similar things can happen with the call to the dispatcher. The top frame
-// may not be the primitive. Likewise the primitive can have fewer stack frames
-// such as when a call to useState got inlined to use dispatcher.useState.
+// may not be the primitive.
```

Comment:
I don't see how this can happen since the dispatcher is private. Without the "wrapper -> dispatcher" assumption I'm out of ideas on how to enable arbitrary wrapper names like `useHostTransitionStatus` is trying to enable. 

It also sounds like it contradicts the current implementation of `findPrimitiveIndex` where we always assumed there are two stack frames for the dispatcher method and the public method right next to each other.